ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -376,7 +396,7 @@ def test_det(
 
 # eigh
 @given(
-    dtype_x=_get_dtype_and_matrix(symmetric=True),
+    dtype_x=_get_dtype_and_symm_matrix(),
```

Comment:
please don't create a new helper, given that `_get_dtype_and_matrix` already supports symmetric matrices. You're welcome to modify the implementation of this if you want, but if you do the `(x + x.T)/2` approach then this should be done in NumPy rather than Ivy.

Reply:
Oh okay sure @djl11 it seems like this functionality was added after I had already implemented my function. I didn't want to clutter the first function so I created another one, but the new approach is great too! Will make these changes.